autoscheduler/frontend/src/components/SchedulingPage/Schedule/Schedule.css

```diff
@@ -1,11 +1,12 @@
 @value v-time-label-margin: 48px;
+@value fullscreen-row-height: 48px;
 
 .calendar-container {
     display: flex;
     flex-direction: column;
     padding: 8px;
     font-family: "Roboto", "Helvetica", "Arial", sans-serif;
-    height: calc(100% - 24px);
+    height: calc(100% - 24px - fullscreen-row-height);
```

Comment:
What's preventing you from making the parent element flex rather than manually subtracting the fullscreen row height?

Reply:
Good call - I honestly thought I had done this and it didn't work but I guess I did something wrong the first time.